pkg/cli/token_create.go

```diff
@@ -125,7 +134,22 @@ func (c *TokenCreateCommand) Flags() *cli.FlagSet {
 		Target:  &c.flagExplanation,
 		Aliases: []string{"e"},
 		Example: "Debugging ticket #123",
-		Usage:   `A reason for the action.`,
+		Usage:   `DEPRECATED. Use "justification" flag instead. This flag will be removed in a future release.`,
```

Comment:
I'd probably leave the message as-is and hide the flag: https://github.com/abcxyz/pkg/blob/05a9df35d989594e4f242e397f29d96f3fc0b041/cli/flags.go#L542

```suggestion
Hidden: true,
Usage: `A reason for the action.`,
```